pilot/pkg/xds/discovery.go

```diff
@@ -329,9 +329,7 @@ func (s *DiscoveryServer) Push(req *model.PushRequest) {
 	if err != nil {
 		return
 	}
-
-	initContextTime := time.Since(t0)
-	log.Debugf("InitContext %v for push took %s", versionLocal, initContextTime)
```

Comment:
Can we keep the log? Metrics do not fully replace logs as they are aggregated

Reply:
Sure